.github/workflows/redhat-certify.yml

```
@@ -28,45 +28,24 @@ permissions:
-          echo "=== Downloaded license files ==="
-          ls -la licenses/third-party/ 2>/dev/null | head -20 || echo "No third-party dir found!"
-          echo "Total license files: $(ls licenses/third-party/ 2>/dev/null | wc -l)"
-
+      # Certification runs against the image that was ALREADY published and signed
+      # by ci.yml on the release tag. It deliberately does NOT rebuild: a rebuild
+      # produces a different digest, is not covered by the release's cosign
+      # signature or SLSA attestation, and pushing it over :<version> and :latest
+      # would replace what users pull with unsigned bytes.
       - name: Set image coordinates
         id: image
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:3:26: Double quote to prevent globbing and word splitting

(shellcheck)